Update docker.md Nginx section (#2021)
saucelabs/sauce-docs: docs/secure-connections/sauce-connect/setup-configuration/docker.md

}
```

## Running the Sauce Connect Docker with a nginx Server

> The `docker-compose.yaml` file below shows how to run Sauce Connect Docker in the nginx server. This is an example where you can use the tunnel while launching a Desktop or Web Mobile test, then go to the url 'nginx'. Sauce Connect Proxy will route you to the nginx container!

Reviewer: @maxdobeck Might this work?

> The `docker-compose.yaml` file below shows how to run Sauce Connect Docker in the nginx server. This is an example where you can use the tunnel while launching a Desktop or Web Mobile test, then go to the url 'nginx'. Sauce Connect Proxy will route you to the nginx container!

Reviewer 2: ```suggestion
The `docker-compose.yaml` file below shows how to run Sauce Connect Docker in the nginx server. The example configuration allows you to use a tunnel while launching a Desktop or Web Mobile test, then go to the 'nginx' URL where Sauce Connect Proxy routes you to the nginx container.
```
